composer.json

```diff
@@ -132,15 +132,15 @@
         "ext-fileinfo": "*",
         "composer/composer": "^1.0",
         "contao/monorepo-tools": "dev-master",
-        "contao/phpstan": "^0.4",
+        "contao/phpstan": "^0.5",
         "contao/test-case": "^2.0",
         "doctrine/doctrine-migrations-bundle": "^1.1",
         "doctrine/event-manager": "^1.0",
         "doctrine/orm": "^2.5",
         "friendsofphp/php-cs-fixer": "^2.12",
         "monolog/monolog": "^1.22",
         "php-coveralls/php-coveralls": "^2.1",
-        "phpstan/phpstan-phpunit": "^0.10",
+        "phpstan/phpstan-phpunit": "^0.11",
```

Comment:
Why is this necessary?

Reply:
https://getcomposer.org/doc/articles/versions.md#caret-version-range-

> The ^ operator behaves very similarly but it sticks closer to semantic versioning, and will always allow non-breaking updates. For example ^1.2.3 is equivalent to >=1.2.3 <2.0.0 as none of the releases until 2.0 should break backwards compatibility. For pre-1.0 versions it also acts with safety in mind and treats ^0.3 as >=0.3.0 <0.4.0.